client/src/components/Form/FormMessage.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+    <b-alert :variant="variant" :show="showAlert">
+        {{ message }}
+    </b-alert>
+</template>
+<script>
+import Vue from "vue";
```

Comment:
If you use the standard mount you do not need to import any of this every time.

Reply:
Done.